Assets/MixedRealityToolkit/EventSystem/EventManager.cs

```diff
@@ -21,7 +24,7 @@ public class EventManager : BaseManager, IEventSystemManager
         /// <param name="eventHandler">Event Handler delegate</param>
         public virtual void HandleEvent<T>(BaseEventData eventData, ExecuteEvents.EventFunction<T> eventHandler) where T : IEventSystemHandler
         {
-            Debug.Assert(!eventData.used);
+             Debug.Assert(!eventData.used);
```

Comment:
is this assert right? shouldn't eventData.used == true just cause an early exit?